dvc/analytics.py

```diff
@@ -1,256 +1,158 @@
-"""Collect and send usage analytics"""
-from __future__ import unicode_literals
```

Comment:
Probably shouldn't delete this one :)

Reply:
Not using unicode on the analytics module, but I can add it back.

Reply:
done)